RN Maps is imported into Jigsaw but is not supported on web. 
Since all 'require' statements are parsed/analysed this creates an issue due to missing native packages that are imported
This patches RN Maps to include an 'empty' web version that removes reliance on imports not available on web

Fix as sugegsted here: https://github.com/react-native-maps/react-native-maps/issues/4641
Bigger, general expo issue: https://github.com/expo/expo/issues/16513

diff --git a/node_modules/react-native-maps/lib/index.web.d.ts b/node_modules/react-native-maps/lib/index.web.d.ts
new file mode 100644
index 0000000..6ff26b8
--- /dev/null
+++ b/node_modules/react-native-maps/lib/index.web.d.ts
@@ -0,0 +1,37 @@
+import MapView, { AnimatedMapView as Animated, MAP_TYPES, enableLatestRenderer, MapViewProps } from 'react-native-maps/lib/MapView';
+import Marker from 'react-native-maps/lib/MapMarker';
+export { MapMarker } from 'react-native-maps/lib/MapMarker';
+export type { MapMarkerProps } from 'react-native-maps/lib/MapMarker';
+import Overlay from 'react-native-maps/lib/MapOverlay';
+export { MapOverlay } from 'react-native-maps/lib/MapOverlay';
+export type { MapOverlayProps } from 'react-native-maps/lib/MapOverlay';
+export { default as Polyline, MapPolyline } from 'react-native-maps/lib/MapPolyline';
+export type { MapPolylineProps } from 'react-native-maps/lib/MapPolyline';
+export { default as Heatmap, MapHeatmap } from 'react-native-maps/lib/MapHeatmap';
+export type { MapHeatmapProps } from 'react-native-maps/lib/MapHeatmap';
+export { default as Polygon, MapPolygon } from 'react-native-maps/lib/MapPolygon';
+export type { MapPolygonProps } from 'react-native-maps/lib/MapPolygon';
+export { default as Circle, MapCircle } from 'react-native-maps/lib/MapCircle';
+export type { MapCircleProps } from 'react-native-maps/lib/MapCircle';
+export { default as UrlTile, MapUrlTile } from 'react-native-maps/lib/MapUrlTile';
+export type { MapUrlTileProps } from 'react-native-maps/lib/MapUrlTile';
+export { default as WMSTile, MapWMSTile } from 'react-native-maps/lib/MapWMSTile';
+export type { MapWMSTileProps } from 'react-native-maps/lib/MapWMSTile';
+export { default as LocalTile, MapLocalTile } from 'react-native-maps/lib/MapLocalTile';
+export type { MapLocalTileProps } from 'react-native-maps/lib/MapLocalTile';
+export { default as Callout, MapCallout } from 'react-native-maps/lib/MapCallout';
+export type { MapCalloutProps } from 'react-native-maps/lib/MapCallout';
+export { default as CalloutSubview, MapCalloutSubview, } from 'react-native-maps/lib/MapCalloutSubview';
+export type { MapCalloutSubviewProps } from 'react-native-maps/lib/MapCalloutSubview';
+export { default as AnimatedRegion } from 'react-native-maps/lib/AnimatedRegion';
+export { default as Geojson } from 'react-native-maps/lib/Geojson';
+export type { GeojsonProps } from 'react-native-maps/lib/Geojson';
+export { Marker, Overlay };
+export type { MapViewProps };
+export { Animated, MAP_TYPES, enableLatestRenderer };
+export * from 'react-native-maps/lib/ProviderConstants';
+export * from 'react-native-maps/lib/MapView.types';
+export * from 'react-native-maps/lib/sharedTypes';
+export declare const MarkerAnimated: import("react-native").Animated.AnimatedComponent<typeof import("react-native-maps/lib/MapMarker").MapMarker>;
+export declare const OverlayAnimated: import("react-native").Animated.AnimatedComponent<typeof import("react-native-maps/lib/MapOverlay").MapOverlay>;
+export default MapView;
diff --git a/node_modules/react-native-maps/lib/index.web.js b/node_modules/react-native-maps/lib/index.web.js
new file mode 100644
index 0000000..7c6d6c7
--- /dev/null
+++ b/node_modules/react-native-maps/lib/index.web.js
@@ -0,0 +1 @@
+module.exports = {}
\ No newline at end of file
diff --git a/node_modules/react-native-maps/package.json b/node_modules/react-native-maps/package.json
index 873c1a7..8613a96 100644
--- a/node_modules/react-native-maps/package.json
+++ b/node_modules/react-native-maps/package.json
@@ -2,7 +2,8 @@
   "name": "react-native-maps",
   "description": "React Native Mapview component for iOS + Android",
   "source": "src/index",
-  "main": "lib/index.js",
+  "main": "lib/index.web.js",
+  "react-native": "lib/index.js",
   "author": "Leland Richardson <leland.m.richardson@gmail.com>",
   "homepage": "https://github.com/react-native-maps/react-native-maps#readme",
   "version": "1.3.2",
